Source/Core/VideoCommon/FrameDump.h

```diff
@@ -22,7 +22,7 @@ class FrameDump
     int savestate_index = 0;
   };
 
-  static bool Start(int w, int h);
+  static bool Start(int w, int h, bool new_segment = false);
```

Comment:
This should likely be an enum rather than a bool so it's obvious at the callsite what behavior is being signified.